terraform/modules/cluster/mig-with-container/cloudinit/templates/userdata_gpu.yaml.template

```diff
@@ -0,0 +1,33 @@
+# cloud-config
+
+write_files:
+- path: /etc/systemd/system/aiinfra_install_gpu.sh
+  permissions: 0755
+  owner: root
+  content: |
+    #!/bin/sh
+    cos-extensions install gpu
```

Comment:
we need to run `cos-extensions install gpu -- --version=latest` if vm type is A3.